WellnessLiving/Wl/Location/View/ViewModel.php

```diff
@@ -309,6 +309,13 @@ class ViewModel extends WlModelAbstract
    */
   public $url_site = '';
 
+    /**
+     * Twitter url.

```

Comment:
Which twitter URL? Probably the URL of the location's twitter account, it should be said.

Reply:
Copied that over from the corresponding API block. Now fixed.